### ISSUE TITLE
AKD mega issue

### ISSUE THREAD

**Issue by Hydrox6** [runelite/runelite#13745]
**New Clues:** :heavy_check_mark:
<details><summary>Clues info</summary>
We know there are the following clues that are not accounted for:
* 2 Easy clues
* 3 Medium clues, 1 Challenge scroll
* 3 Hard clues
* 2 Elite clues

There may also be Master steps we don't know about. Hot/Cold steps should already all be accounted for.

---

**Easy Clues:**
<details><summary>Hosidius</summary>
<img src=https://user-images.githubusercontent.com/2979691/122249494-cfc88600-cec0-11eb-883b-7d6f2f0f4bc8.png>
<img src=https://user-images.githubusercontent.com/2979691/122249971-2fbf2c80-cec1-11eb-9764-f761464aec43.png>

</details>

---

**Medium Clues:**
<details><summary>Shayzien</summary>
<img src=https://user-images.githubusercontent.com/2979691/122242815-8590d600-cebb-11eb-9ddb-b23ccd9f2c8f.png>

Solution requires killing a hill giant for a key, then opening a crate in Osten's shop
</details>

<details><summary>Shayzien</summary>
<a href=https://imgur.com/a/aMzZ20F>Imgur Album</a>

Solution is a drunken soldier in Shayzien, challenge scroll answer is 598
</details>

<details><summary>Hosidius</summary>
<img src=https://user-images.githubusercontent.com/2979691/122287587-01554780-cee9-11eb-9aff-8263eae6938d.png>
</details>

---

**Hard Clues:**
<details><summary>Forthos Dungeon</summary>
<img src=https://user-images.githubusercontent.com/2979691/122241760-b6244000-ceba-11eb-9744-976787c9925c.png>
<img src=https://user-images.githubusercontent.com/2979691/122241731-b290b900-ceba-11eb-8fa5-960799fe66cd.png>
</details>

<details><summary>Music clue (no changes needed)</summary>
<img src=https://user-images.githubusercontent.com/2979691/122276926-33f94300-cedd-11eb-930d-529f0955ee9e.png>
</details>

---

**Elite Clues:**
<details><summary>Hosidius</summary>
<img src=https://user-images.githubusercontent.com/5871961/122269167-879a6b00-cea2-11eb-91e8-d3b329539c4e.png>
</details>

---

In addition, a few other clues have broken or need some update:
<details><summary>Broken clue</summary>
<img src=https://user-images.githubusercontent.com/2979691/122244843-1caa5d80-cebd-11eb-96e0-db7824feaf23.png>
This clue seems to have had very minor text changes which cause it to be unrecognised.
</details>
<details><summary>Shayzien clues</summary>
"Captain Ginea" anagram clue needs a location update

"Beckon in the combat ring of Shayzien." emote clue needs a location update, along with its STASH

"Shrug in the Shayzien command tent." emote clue needs a location update, along with its STASH

"New recruit Tony" anagram clue needs a location update

"Graveyard of Heroes" clue could do with an updated hint for travel
</details>
<details><summary>Incorrect Hot/Cold enemy</summary>
"East of Watson's house" hot/cold should be listed as having a Brassican mage
</details>
</details>

---

**Timers** :heavy_check_mark:
<details><summary>Clues info</summary>
Basically all of the updated Arceuus spellbook has spells with timers of some form. These should be added to the Timer plugin in a similar way to existing spells (eg. Charge)
</details>

---

**Skill Calculator** :heavy_check_mark:

<details><summary>Skill Calculator</summary>
The new Arceuus spellbook has some skilling spells that would fit into the skill calculator.
The new ashes should be added too.
</details>

**Comment by Nightfirecat** [comment]
I don't have the various NPC names and object IDs needed for some of the added clues, but I do have the coordinates. I have that incomplete data here: https://github.com/Nightfirecat/runelite/commit/9297de8c2dac2119b98e130c6ff9a74b5cd7545d

**Comment by makingcomment** [comment]
One of the hard clues is a Cecelia clue for the song Regal Pomp. [(source)](http://redd.it/o12bgk)

**Comment by andmcadams** [comment]
![image](https://user-images.githubusercontent.com/5871961/122269167-879a6b00-cea2-11eb-91e8-d3b329539c4e.png)

This is one of the elite clues

**Comment by Nightfirecat** [comment]
All these items are addressed in #13744 & #13749 (clues), #13750 (timers), and #13762 (skill calculator)
